fastlane/lib/fastlane/actions/get_version_number.rb

```diff
@@ -39,9 +40,14 @@ def self.run(params)
         return version_number
       end
 
-      def self.get_project!(folder)
+      def self.get_project!(xcodeproj_path_or_dir)
         require 'xcodeproj'
-        project_path = Dir.glob("#{folder}/*.xcodeproj").first
+        if File.extname(xcodeproj_path_or_dir) == ".xcodeproj"
+          project_path = xcodeproj_path_or_dir
+        else
+          project_path = Dir.glob("#{xcodeproj_path_or_dir}/*.xcodeproj").first
+        end
+
         if project_path
```

Comment:
```suggestion
        if project_path && File.exist?(project_path)
```

This more makes sense? This mitigates the concern minuscorp has below.

Reply:
Makes sense! 😊